binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -128,6 +131,14 @@ public void run() {
                                 }
                             }
                         }
+                        
+                        final Config config = fullConfig.getConfig(); 
+                        if (config != null) {
+                            getThing().getConfiguration().put(ThingProperty.SERIAL_NUMBER.keyName,
```

Comment:
You are executing the code on every single polling request. It should instead only be done once on the first successful connection. Or maybe not even then? Maybe only on first connection if the thing does not yet have the properties set?

I've also discussed with @dnobel that we will need a mechanism to tell the system that a thing has been updated and needs to be persisted. This would have to be used here, once it is available.